internal/boxcli/cache.go

```diff
@@ -103,6 +108,14 @@ func cacheCredentialsCmd() *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			if flags.format == "env" {
```

Comment:
Just to future-proof ourselves, what about "sh" (following brew and shellcheck's convention) to differentiate from fish or other shells.